Entity/SelectedStopPointRepository.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace CanalTP\MttBundle\Entity;
+
+use Doctrine\ORM\EntityRepository;
+
+/**
+ * SelectedStopPointRepository
+ *
+ * This class was generated by the Doctrine ORM. Add your own custom
+ * repository methods below.
+ */
+class SelectedStopPointRepository extends EntityRepository
```

Comment:
A custom repository is usefull if you create custom methods.

If they are empty you don't need them. Doctrine is magic.